gef.py

```diff
@@ -3621,21 +3621,26 @@ def parse_string_range(s):
 
 @lru_cache()
 def gef_get_auxiliary_values():
-    """Retrieves the auxiliary values of the current execution. Returns None if not running, or a dict()
-    of values."""
+    """Retrieves the ELF auxiliary values of the current execution. This information is provided by
```

Comment:
Can you please use editor wrapping on this. `gq<left>` in vim.

Reply:
Nit, I know not your doc string but we can fix.

Imperative tone, like you're commanding the function: "Retrieve..." and "Return...".

Reply:
done